slack-archive/src/main/java/juja/microservices/slack/archive/controller/ArchiveController.java

```diff
@@ -33,8 +40,9 @@ public void archiveSlackMessage(@RequestParam("token") String token,
 
     @GetMapping(value = "/channels", consumes = "application/json")
     public ResponseEntity<?> getChannels() {
-        //TODO Should be implemented
-        return null;
+        log.info("Received request to get all channels");
```

Comment:
Maybe INFO log place at the end? With the short result?

Reply:
Yes, I fixed it.